python_ta/checkers/infinite_loop_checker.py

```diff
@@ -0,0 +1,73 @@
+"""Check for infinite while loops."""
+
+from typing import Optional
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.lint import PyLinter
+
+
+class InfiniteLoopChecker(BaseChecker):
+    name = "infinite-loop"
+    msgs = {
+        "W9501": (
+            "No condition variable in the while loop is used inside the loop body",
+            "loop-condition-variable-unused",
```

Comment:
You can just call this check `infinite-loop`. Even though the implementation is only detecting one case currently, we can extend it to detect others in the future, so we shouldn't limit ourselves to a particular case in the error name/message/description. Please update the message and description as well.